spec/super_good/solidus_taxjar/backfill_transactions_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "spec_helper"
+
+RSpec.describe SuperGood::SolidusTaxjar::BackfillTransactions do
+  describe ".call" do
+    subject { described_class.new(api: api_spy).call }
+
+    let!(:shipped_order) { create :shipped_order }
+    let(:api_spy) { instance_spy(::SuperGood::SolidusTaxjar::Api) }
+
+    around do |example|
+      ::SuperGood::SolidusTaxjar.test_mode = true
+      example.run
+      ::SuperGood::SolidusTaxjar.test_mode = false
+    end
+
+    before do
+      reported_order = create :shipped_order
+      create(:taxjar_order_transaction, order: reported_order)
+      create :completed_order_with_totals
+      create :order_ready_to_ship
+      create :order
```

Comment:
I don't think we need to create all of these orders. We can't have a shipped incomplete order so just a single order here should suffice to test that condition.

Reply:
Addressed!